### ISSUE TITLE
Renal cortical hyperechogenicity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6198]
**Preferred term label:**
Renal cortical hyperechogenicity

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/17663471/

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0004719

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/17663471/  4/4 OMIM:611555, others

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP_0033132
